go.mod

```diff
@@ -1,43 +1,33 @@
-module github.com/justinbarrick/go-k8s-portforward
+module github.com/2opremio/go-k8s-portforward
```

Comment:
I'm going to have to fix this.

Reply:
Thanks, I kept pushing to my master not thinking about the side-effects on this PR.